### PR TITLE
chore(core): Stop reporting to Sentry execution cancellations

### PR DESCRIPTION
**Pull request by ivov**
https://linear.app/n8n/issue/PAY-2139

### CODE REVIEW

**Review by netroy — COMMENTED** [review]
can we add a check for `ExecutionCancelledError` in `beforeSend` instead ?

**Comment by ivov** [comment]
My understanding (not 100%) is that `ExecutionCancelledError` is already listed as `warning` but here we're throwing `InternalServerError` (reusing the message) which is causing the Sentry report.

Our error setup needs improving so that we don't have to account for this, but for now let's stop reporting it?

File: packages/cli/src/webhooks/webhook-helpers.ts [hunk 1]
```diff
@@ -756,7 +757,9 @@ export async function executeWebhook(
 						);
 					}
 
-					throw new InternalServerError(e.message);
+					const internalServerError = new InternalServerError(e.message);
```
Comment (netroy) [inline packages/cli/src/webhooks/webhook-helpers.ts:760]: we really should start adding the original error as `cause` in all instances of `ResponseError`, and then update `beforeSend` to either check `if (error instanceof ResponseError && error.cause instanceof ExecutionCancelledError)` or `if (error instanceof ResponseError && error.cause.level === 'warning')`, to make sure that we don't have to sprinkle checks like these across the codebase.

**Review by netroy — APPROVED** [review]

File: packages/cli/src/webhooks/webhook-helpers.ts (hunk 1 above)
Comment (ivov) [reply packages/cli/src/webhooks/webhook-helpers.ts:760]: Definitely agree, tracking here: https://linear.app/n8n/issue/PAY-2145

**Comment by janober** [comment]
Got released with [`n8n@1.65.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.65.0)

**Pull request merged**